docs/source/release_notes/release-notes.rst

```diff
@@ -42,6 +42,39 @@ develop
     *    - Type
          - Change
 
+    *    -
+         -
+
+
+======
+0.45.0
+======
+
+19 June 2017
+
+.. list-table::
+    :widths: 5 40
+    :header-rows: 1
+
+    *    - Type
+         - Change
+
+    *    - |devbreak| |improved|
+         - Upgraded all usages of http-remoting to remoting2.
```

Comment:
Yep, added. I also mentioned what would be most likely to break (`TM.create()`).